cypress/support/util/AdminClient.ts

```diff
@@ -184,6 +184,57 @@ class AdminClient {
     await this.login();
     return await this.client.roles.delByName({ name });
   }
+
+  async createIdentityProvider(idpDisplayName: string, alias: string) {
+    await this.login();
+    const identityProviders =
+      (await this.client.serverInfo.find()).identityProviders || [];
+    const idp = identityProviders.find((obj) => {
+      return obj.name === idpDisplayName;
+    });
```

Comment:
Can be cleaned up by using destructuring and implicit returns.
```suggestion
    const idp = identityProviders.find(({ name }) => name === idpDisplayName);
```